distribution/src/main/assembly/dist.xml

```diff
@@ -246,7 +246,7 @@
             </excludes>
         </fileSet>
         <fileSet>
-            <directory>../documentation/manual/target/docbook/publish</directory>
+            <directory>../documentation/manual/target/asciidoctor</directory>
```

Comment:
Yeah, I must admit I'm a bit reluctant to do last minute changes that can break something when they are not strictly necessary as I tested the whole chain for all the projects and I'd like to keep them consistent.